.travis.yml

```diff
@@ -23,3 +23,5 @@ env:
   - DB=mysql
 rvm:
   - 2.1
+  - 2.2
+  - 2.3
```

Comment:
It must be specified as `2.3.0` (see https://github.com/refinery/refinerycms/blob/dc0ee89a651dd8fb6755230ea65ce933d8468d1a/.travis.yml#L31)